packages/web-components/src/components/content-item-horizontal/content-item-horizontal-thumbnail.scss

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright IBM Corp. 2020, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// @import '@carbon/ibmdotcom-styles/scss/components/content-item-horizontal-thumbnail/content-item-horizontal-thumbnail';
+
+@import '@carbon/ibmdotcom-styles/scss/globals/imports';
+@import '@carbon/ibmdotcom-styles/scss/components/link-list/index';
+
+@mixin content-item-horizontal-thumbnail {
+  // Container
+  :host(#{$dds-prefix}-content-item-horizontal-thumbnail) {
+    @include carbon--make-row;
+
+    border-top: 1px solid $ui-03;
+    margin: 0;
+    padding-left: $carbon--spacing-05;
+    padding-right: $carbon--spacing-05;
+
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    @include carbon--breakpoint(md) {
+      min-height: rem(306px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-10;
+    }
+
+    @include carbon--breakpoint(lg) {
+      min-height: rem(272px);
+    }
+
+    @include carbon--breakpoint(xlg) {
+      min-height: rem(252px);
+    }
+  }
+
+  // Layout
+  .#{$prefix}--content-item-horizontal-thumbnail__row {
+    @include carbon--make-row;
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__col {
+    @include carbon--make-col-ready;
+
+    &--1 {
+      @include carbon--breakpoint(sm) {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint(md) {
+        @include carbon--make-col(4, 8);
+      }
+
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(8, 12);
+        @include carbon--make-row;
+
+        margin: 0;
+      }
+    }
+
+    &--2 {
+      @include carbon--breakpoint(sm) {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint(md) {
+        @include carbon--make-col(4, 8);
+
+        display: flex;
+        justify-content: flex-end;
+      }
+
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 12);
+
+        display: flex;
+      }
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__heading-wrapper,
+  .#{$prefix}--content-item-horizontal-thumbnail__content-wrapper {
+    @include carbon--breakpoint(md) {
+      padding-right: $carbon--spacing-07;
+    }
+    @include carbon--breakpoint(lg) {
+      @include carbon--make-col(4, 8);
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__heading-wrapper {
```

Comment:
instead of using a wrapper class, could you target the heading with the `:host(#{$dds-prefix}-content-item-horizontal-thumbnail) ::slotted([slot='heading'])` selector that you have below?